noetic/distribution.yaml

```diff
@@ -9706,6 +9706,23 @@ repositories:
       url: https://gitlab.com/toposens/public/ros-packages.git
       version: master
     status: developed
+  toposens-library:
+    doc:
+      type: git
+      url: https://gitlab.com/toposens/public/toposens-library.git
```

Comment:
The license specified in the package.xml doesn't match the one specified in the top level license file.
Please, update either as appropriate to be consistent.

Reply:
Thanks, I updated it.